File: uri/ad-hock/libertadores/1536.cpp
#include <iostream>

using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int t;
    if (!(cin >> t)) {
        return 0;
    }
    while (t--) {
        int m1, v1, m2, v2;
        char sep;
        cin >> m1 >> sep >> v1;
        cin >> m2 >> sep >> v2;
        int points1 = 0, points2 = 0;
        if (m1 > v1) {
            points1 += 3;
        } else if (m1 < v1) {
            points2 += 3;
        } else {
            points1 += 1;
            points2 += 1;
        }
        if (m2 > v2) {
            points2 += 3;
        } else if (m2 < v2) {
            points1 += 3;
        } else {
            points1 += 1;
            points2 += 1;
        }
        int goals1 = m1 + v2;
        int goals2 = v1 + m2;
        int diff1 = goals1 - goals2;
        int diff2 = -diff1;
        int away1 = v2;
        int away2 = v1;
        if (points1 > points2) {
            cout << "Time 1\n";
        } else if (points2 > points1) {
            cout << "Time 2\n";
        } else if (diff1 > diff2) {
            cout << "Time 1\n";
        } else if (diff2 > diff1) {
            cout << "Time 2\n";
        } else if (away1 > away2) {
            cout << "Time 1\n";
        } else if (away2 > away1) {
            cout << "Time 2\n";
        } else {
            cout << "Penaltis\n";
        }
    }
    return 0;
}

File: uri/ad-hock/libertadores/1536.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let mut iter = input.split_whitespace();
    let t: i32 = match iter.next() {
        Some(v) => v.parse().unwrap(),
        None => return,
    };
    for _ in 0..t {
        let m1: i32 = iter.next().unwrap().parse().unwrap();
        iter.next();
        let v1: i32 = iter.next().unwrap().parse().unwrap();
        let m2: i32 = iter.next().unwrap().parse().unwrap();
        iter.next();
        let v2: i32 = iter.next().unwrap().parse().unwrap();
        let mut points1 = 0;
        let mut points2 = 0;
        if m1 > v1 {
            points1 += 3;
        } else if m1 < v1 {
            points2 += 3;
        } else {
            points1 += 1;
            points2 += 1;
        }
        if m2 > v2 {
            points2 += 3;
        } else if m2 < v2 {
            points1 += 3;
        } else {
            points1 += 1;
            points2 += 1;
        }
        let goals1 = m1 + v2;
        let goals2 = v1 + m2;
        let diff = goals1 - goals2;
        let away1 = v2;
        let away2 = v1;
        if points1 > points2 {
            println!("Time 1");
        } else if points2 > points1 {
            println!("Time 2");
        } else if diff > 0 {
            println!("Time 1");
        } else if diff < 0 {
            println!("Time 2");
        } else if away1 > away2 {
            println!("Time 1");
        } else if away2 > away1 {
            println!("Time 2");
        } else {
            println!("Penaltis");
        }
    }
}
